### PR TITLE
Feature/gh 160 Include "Add all Terms" option to multi-term selector

### LINKED ISSUE #160: 'Select all' option when adding terms to a codeset
When choosing terms from a terminology to add to a codeset, it would be nice if there was a 'select all' option.  There are similar components where we add types to a model, etc - would be nice to ensure we have that option everywhere.

### PR DESCRIPTION
**Pull request by pjmonks**
Resolves #160 

Update `mdm-multiple-terms-selector` component to include an "Add all Terms" option:

image

If ticked, then every term in the selected terminology will be added to the parent code set. Untick to revert back to the manual search/filter selection list to pick terms.

### CODE REVIEW

File: src/app/utility/multiple-terms-selector/multiple-terms-selector.component.html [hunk 1]
```diff
@@ -17,74 +17,156 @@
...
+      <!-- <div style="height: 400px; "> -->
+      <mat-table
+        #table
+        [dataSource]="dataSource"
+        style="max-height: 100%; overflow: auto"
+      >
+        <ng-container matColumnDef="label">
+          <mat-header-row *matHeaderCellDef> Term </mat-header-row>
+          <mat-cell
+            style="
```
Comment (gammonpeter) [inline src/app/utility/multiple-terms-selector/multiple-terms-selector.component.html:71]: Might be better placed in class

File: src/app/utility/multiple-terms-selector/multiple-terms-selector.component.ts [hunk 2]
```diff
@@ -126,9 +129,11 @@ export class MultipleTermsSelectorComponent {
       this.currentRecord = 0;
     }
   };
+
   runTermSearch = () => {
```
Comment (gammonpeter) [inline src/app/utility/multiple-terms-selector/multiple-terms-selector.component.ts:133]: Might be good time to refactor this

**Review by gammonpeter — CHANGES_REQUESTED** [review]
Not sure the checked box should default to true. I think it should be a positive choice to select all not one that's made for me.

File: src/app/utility/multiple-terms-selector/multiple-terms-selector.component.html (hunk 1 above)
Comment (pjmonks) [reply src/app/utility/multiple-terms-selector/multiple-terms-selector.component.html]: Done

File: src/app/utility/multiple-terms-selector/multiple-terms-selector.component.ts (hunk 2 above)
Comment (pjmonks) [reply src/app/utility/multiple-terms-selector/multiple-terms-selector.component.ts]: Done

**Review by gammonpeter — APPROVED** [review]

**Pull request merged**